cdap-ui/app/features/etlapps/controllers/create-ctrl.js

```diff
@@ -196,6 +196,19 @@ angular.module(PKG.name + '.feature.etlapps')
       });
     };
 
+    $scope.deleteTrasformProperty = function(transform) {
+      var filterFilter = $filter('filter');
+      var index = $scope.transforms.indexOf(transform);
+      $scope.transforms.splice(index, 1);
```

Comment:
We are assuming that we can always find the element. Is there a case where we might not find a match when clicking delete (ie. syncing problem)?